en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -1,170 +1,170 @@
-一、打开主域项目，在 `菜单栏` - `项目` 中打开构建发布面板，选择 `Wechat Game` 平台，填入 [开放数据域代码目录]。该目录是开放数据域构建后所在的路径，并且这个路径需要放在主域构建目录下。然后点击 **构建**。
+1. Open the main domain project, open the build release panel in `Menu Bar` - `Project`, select the `Wechat Game` platform, and fill in the [Open Data Field Code Directory]. This directory is the path where the open data domain is built, and this path needs to be placed in the main domain build directory. Then click on **Build**.
 
 ![](./publish-wechatgame/maintest-build.png)
 
-该步骤会帮用户自动配置到主域项目 `build -> wechatgame -> game.json` 中，用于辨别开放数据域文件在主域发布包下的所在目录。
+This step will automatically configure the user to the main domain project `build -> wechatgame -> game.json` to identify the directory where the open data domain file is located under the main domain distribution package.
 
 ![](./publish-wechatgame/game-json.png)
 
-二、打开开放数据域项目，打开构建发布面板，选择 `Wechat Game Open Data Context` 平台。
+Second, open the open data domain project, open the build release panel, select the `Wechat Game Open Data Context` platform.
 
-三、**发布路径** 设置为主域中填入的 [开放数据域代码目录] 相同路径，即指定到主域项目工程的发布包目录下。然后点击 **构建**。
+Third, the **release path ** set the same path in the [open data domain code directory] filled in the main domain, that is, assigned to the release package directory of the main domain project. Then click on **Build**.
 
-**注意** ：**游戏名称** 必须和主域项目中设置的 [开放数据域代码目录] 名称一致。
+**Note**: **Game Name** must match the [Open Data Field Code Directory] name set in the main domain project.
 
 ![](./publish-wechatgame/open-data-project-build.png)
 
-或者可以不修改 **发布路径**，在开放数据域项目构建完成后手动将发布包拷贝到主域项目的发布包目录下。如下图所示：
+Or you can modify the **release path** to manually copy the release package to the release package directory of the primary domain project after the open data domain project is built. As shown below:
 
 ![](./publish-wechatgame/package.png)
 
-四、在主域项目工程中点击 [运行] 调起微信开发者工具，即可按照之前微信小游戏的正常流程进行发布和调试。
+4. Click [Run] in the main domain project to launch the WeChat Developer tool, and then publish and debug according to the normal process of the previous WeChat game.
 
 ![](./publish-wechatgame/preview.png)
 
-**注意：如果先发布开放数据域域再发布主域，那么开放数据域的发布代码会被覆盖，我们会在后续版本优化体验**
+**Note: If you publish the open data domain domain and then publish the primary domain, the release code of the open data domain will be overwritten, and we will optimize the experience in subsequent versions**
 
-## 参考链接
+## Reference link
 
-Cocos Creator 提供的 [开放数据域范例工程](https://github.com/cocos-creator/demo-wechat-subdomain/archive/master.zip) 包含了微信开放数据域的使用示例。
+The [Open Data Domain Sample Project] (https://github.com/cocos-creator/demo-wechat-subdomain/archive/master.zip) provided by Cocos Creator contains examples of the use of WeChat open data fields.
 
-[微信官方文档：关系链数据使用指南](https://developers.weixin.qq.com/minigame/dev/tutorial/open-ability/open-data.html)
+[WeChat official document: Guide to the use of relational chain data] (https://developers.weixin.qq.com/minigame/dev/tutorial/open-ability/open-data.html)
```

Comment:
Guide to the use of relational chain data -> Relationship Chain Data Usage Guide